Sources/MacroCollection/MacroCollection.swift

```diff
@@ -7,12 +7,15 @@
 
 import Foundation
 import SwiftCompilerPlugin
-import SwiftSyntaxMacros
+@_spi(ExperimentalLanguageFeature) import SwiftSyntaxMacros
```

Comment:
Chceme do nasej libky davat veci, ktore su este "Experimental"?

Reply:
zatial to nemergujem, len nechapem preco som toto oznacil ako PR a nie iba ako draft 😄 dakujem za CR, ale este to prehodim do draftu for the time being, kym nevyjde oficialne Swift 6